### PR TITLE
Identity tests #861n34x88

### PR DESCRIPTION
**Pull request by Skava600**

### CODE REVIEW

File: RecordHub.IdentityService/RecordHub.IdentityService.Tests/AccountControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+            var cancellationToken = CancellationToken.None;
+            var model = new LoginModel
+            {
+                UserName = "testuser",
+                Password = "password123"
+            };
+
+            var expectedToken = "validToken";
+            var accountServiceMock = new Mock<IAccountService>();
+            accountServiceMock
```
Comment (Y45K0VL3V) [inline RecordHub.IdentityService/RecordHub.IdentityService.Tests/AccountControllerTests.cs]: All setups put in another file, then you call them one time in constructor

File: RecordHub.IdentityService/RecordHub.IdentityService.Tests/AccountControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                .Setup(m => m.LoginAsync(model, cancellationToken))
+                .ReturnsAsync(expectedToken);
+
+            var controller = new AuthController(accountServiceMock.Object);
+
+            // Act
+            var result = await controller.LoginAsync(model, cancellationToken);
+
+            // Assert
+            var okResult = Assert.IsType<OkObjectResult>(result);
```
Comment (Y45K0VL3V) [inline RecordHub.IdentityService/RecordHub.IdentityService.Tests/AccountControllerTests.cs]: Just use fluent assertions everywhere

File: RecordHub.IdentityService/RecordHub.IdentityService.Tests/AccountControllerTests.cs [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+            var okResult = Assert.IsType<OkResult>(result);
+        }
+
+        [Fact]
+        public async Task UserInfo_AuthorizedUser_ReturnsOkResultWithUserInfo()
+        {
+            // Arrange
+            var cancellationToken = CancellationToken.None;
+            var userId = Guid.Empty;
+            var expectedUser = new UserDTO
```
Comment (Y45K0VL3V) [inline RecordHub.IdentityService/RecordHub.IdentityService.Tests/AccountControllerTests.cs]: this data should be located in another file

**Review by Y45K0VL3V — CHANGES_REQUESTED** [review]
So, just fix everywhere what i marked in the first file (mocks setup, test data, fluent assertions)

File: RecordHub.IdentityService/RecordHub.IdentityService.Tests/AccountControllerTests.cs (hunk 2 above)
Comment (Skava600) [reply RecordHub.IdentityService/RecordHub.IdentityService.Tests/AccountControllerTests.cs]: Should i keep moq assertion? f.e:
   _mockUserManager
       .Verify(m => m
       .AddToRoleAsync(
           It.IsAny<User>(),
           It.IsAny<string>()),
       Times.Once);

Comment (Y45K0VL3V) [reply RecordHub.IdentityService/RecordHub.IdentityService.Tests/AccountControllerTests.cs]: yes

**Review by Y45K0VL3V — APPROVED** [review]

**Pull request merged**